packages/provider-elasticsearch/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contexture-elasticsearch",
-  "version": "1.35.0",
+  "version": "1.36.0",
```

Comment:
@sandhya-spend The `changeset` package manages all the version bumps so we don't need this change.